igvc_description/urdf/jessii.urdf.xacro

```diff
@@ -0,0 +1,655 @@
+                  ixy="0.0"
+                  ixz="0.0"
+                  iyy="${1 / 12 * wheel_mass * wheel_width**2 + 1/4 * wheel_mass * (tire_diameter / 2)**2}"
+                  iyz="0.0"
+                  izz="${1/2 * wheel_mass * (tire_diameter / 2)**2}" />
+      </inertial>
+    </link>
+
+    <xacro:drive_shock lr_prefix="${lr_prefix}" lr_reflect="${lr_reflect}"/>
+
+    <gazebo reference="${lr_prefix}_wheel">
+      <mu1>0.7</mu1>
+      <mu2>0.75</mu2>
+    </gazebo>
+
+  </xacro:macro>
+
+  <!-- macro used to define casters -->
+  <xacro:macro name="caster" params="ball_radius ball_mass tire_diameter x y">
+    <link name="caster">
+      <inertial>
+        <mass value="${ball_mass}" />
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <inertia  ixx="${2 / 5 * ball_mass * ball_radius ** 2}" ixy="0.0" ixz="0.0"
+                  iyy="${2 / 5 * ball_mass * ball_radius ** 2}" iyz="0.0"
+                  izz="${2 / 5 * ball_mass * ball_radius ** 2}" />
+      </inertial>
+      <visual>
+        <geometry>
+          <sphere radius="${ball_radius}"/>
+        </geometry>
+      </visual>
+      <collision>
+        <geometry>
+          <sphere radius="${ball_radius}"/>
+        </geometry>
+      </collision>
+    </link>
+
+    <joint name="base_link_to_caster" type="floating">
+      <parent link="base_link"/>
+      <child link="caster"/>
+      <origin xyz="${x} ${y} ${ball_radius - tire_diameter / 2}" rpy="0 0 0"/>
+    </joint>
+
+    <gazebo reference="caster">
+      <mu1>0.2</mu1>
+      <mu2>0.25</mu2>
+    </gazebo>
+
+    <gazebo>
+      <joint name='base_link_to_caster' type='ball'>
+        <parent>base_link</parent>
+        <child>caster</child>
+        <pose frame=''>0 0 0 0 0 0</pose>
+        <physics>
+          <ode>
+            <limit>
+              <cfm>0</cfm>
+              <erp>0.2</erp>
+            </limit>
+            <suspension>
+              <cfm>0</cfm>
+              <erp>0.2</erp>
+            </suspension>
+          </ode>
+        </physics>
+      </joint>
+    </gazebo>
+  </xacro:macro>
+
+
+  <link name="center_cam">
+      <visual>
+          <geometry>
+              <box size="0.1 0.1 0.1"/>
+          </geometry>
+      </visual>
+  </link>
+
+  <joint name="base_link_to_center_cam" type="fixed">
+      <origin xyz="${center_camera_x} ${center_camera_y} ${center_camera_z}" rpy="0 ${center_camera_pitch} 0" />
+      <parent link="base_link"/>
+      <child link="center_cam"/>
+  </joint>
+
+  <!-- the optical joint is in the correct frame for transforms -->
+  <link name="center_cam_optical"/>
+
+  <joint name="center_cam_optical_joint" type="fixed">
+      <parent link="center_cam"/>
+      <child link="center_cam_optical"/>
+      <origin xyz="0 0 0" rpy="-1.5708 0 -1.5708"/>
+  </joint>
+
+  <gazebo reference="center_cam">
+      <sensor name="center_cam_sensor" type="camera">
+          <camera>
+              <horizontal_fov>1.2290609</horizontal_fov>
```

Comment:
are these values correct for the Logitech C920?

Reply:
nope

Reply:
This was actually right

Reply:
It was in radians. I added a comment